shell.nix

```diff
@@ -11,19 +11,19 @@ in
 
 let
   plutusPkgs = import ./. {};
-  ghc = pkgs.haskell.packages.ghc822.ghcWithPackages (ps: [
-    plutusPkgs.language-plutus-core
-    plutusPkgs.core-to-plc
-    plutusPkgs.plutus-th
-    plutusPkgs.tasty-hedgehog
-    plutusPkgs.tasty
-    plutusPkgs.tasty-golden
-    plutusPkgs.tasty-hunit
-    plutusPkgs.hedgehog
+  ghc = plutusPackages.ghcWithPackages (ps: with ps; [
+    language-plutus-core
+    core-to-plc
+    plutus-th
+    tasty-hedgehog
+    tasty
+    tasty-golden
+    tasty-hunit
+    hedgehog
```

Comment:
I'm still confused about what goes in here, tbh. It has some of the plutus packages only. Should we be using `shellFor` now?